defaults/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+py_version: 2.7.9 # same as AWS
```

Comment:
? I'm not sure if 2.7.9 is the current AWS latest. I logged onto one of our RPC production servers, the Python version in juwai-rpc virtualenv is 2.7.10. (And latest Python 2.7 minor version is 2.7.11).